h2/src/test/org/h2/test/db/TestUpdate.java

```diff
@@ -0,0 +1,82 @@
+package org.h2.test.db;
```

Comment:
Please, move all your tests to `h2/src/test/org/h2/test/scripts/dml/update.sql`. We prefer pure SQL tests where possible.

Reply:
ok. I implemented test cases as sql file.